1-js/11-async/04-promise-error-handling/article.md

```diff
@@ -38,13 +38,13 @@ fetch('/article/promise-chaining/user.json')
 */!*
 ```
 
-Normally, such `.catch` doesn't trigger at all. But if any of the promises above rejects (a network problem or invalid json or whatever), then it would catch it.
+Normalmente, el `.catch` no siempre se dispara. Pero si alguna de las promesas anteriores es rechazada (por un error de red, un JSON inválido or cualquier otra razón), entonces el error es capturado.
```

Comment:
El "Not at all" es terminante: 
"NO en absoluto" o "PARA NADA",  aunque el "Normally" me confunde... 

creo que significa esto  (alguien con más background en  promises podria ayudarnos ???")  "NO, pero..."

```suggestion
Lo normal es que tal `.catch` no se dispare en absoluto. Pero si alguna de las promesas anteriores es rechazada (por un error de red, un JSON inválido or cualquier otra razón), entonces el error es capturado.
```